internal/controllers/clusterextension_controller.go

```diff
@@ -274,7 +269,7 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alp
 		return ctrl.Result{}, err
 	}
 
-	chrt, values, err := r.Handler.Handle(ctx, bundleFS, ext)
+	chrt, values, err := r.Handler.Handle(ctx, bundleFS, bd)
```

Comment:
I assume `bd` for bundleDeployment, but why change and why just this one?

Reply:
This change was made so that we can use the existing handler code in Rukpak and remove the duplicate implementation which we had here (the PR deletes files under handler/). This also makes sure that all the conversion logic from registryV1 to plain is in one place - which is well tested in rukpak, and we have less chances of erroring out.

Reply:
ah I get it, thanks!